src/erlcloud_aws.erl

```diff
@@ -319,8 +319,9 @@ request_to_return(#aws_request{response_type = error,
                                error_type = aws,
                                response_status = Status,
                                response_status_line = StatusLine,
-                               response_body = Body}) ->
-    {error, {http_error, Status, StatusLine, Body}}.
+                               response_body = Body,
+                               response_headers = Headers}) ->
+    {error, {http_error, Status, StatusLine, Body, Headers}}.
```

Comment:
I see changes to the S3 module, but this function is called elsewhere - can you confirm that this doesn't require changes anywhere else?